components/server/src/workspace/workspace-starter.ts

```diff
@@ -308,6 +308,7 @@ export class WorkspaceStarter {
                 project,
                 options.excludeFeatureFlags || [],
                 ideConfig,
+                options.region,
```

Comment:
With this change, after instance creation, there is no need for any further lookup: The instance shape contains everything that is needed to start a workspace.

Only exception that is up for discussion: envVars. Ideally we should store those as well, but I refrained because we might store unencrypted passwords or similar.
Open for comments and ideas here!

Reply:
Is it because JSON stored unencrypted? i.e. how is it different to value in env var tables?